atomic_reactor/util.py

```diff
@@ -597,13 +597,16 @@ def get_build_json():
         raise
 
 
-def is_scratch_build():
-    build_json = get_build_json()
-    try:
-        return build_json['metadata']['labels'].get('scratch', False)
-    except KeyError:
-        logger.error('metadata.labels not found in build json')
-        raise
+def get_build_json_metadata(workflow):
```

Comment:
The problem here is that `get_build_json` is not reliable because `os.environ["BUILD"]` is not always set, right? 

Why not just modify the input plugins to set this environment variable? Or even better, make `get_build_json` take the workflow so it can inspect the result of the `InputPluginsRunner`?